standards/quick-reference.md

```diff
@@ -0,0 +1,22 @@
+# Quick reference
+This page represents a summary of the core standards teams should follow when delivering services for FFC.
+
+These standards are intended to support a consistent interpretation of the Architecture Vision across the programme allowing as much flexibility as possible.
+
+- follow the Architecture Vision
+  - design containerised microservices around business domains and bounded contexts
+  - use asychronous communication between services via Azure Service Bus
+  - orchestrate containers using Kubernetes
+- follow Defra's wider software development standards
+- microservices should be built primarily using Node.js unless there is an evidenced reason not to
+- if a web framework is needed, use Hapi.js
+- if Node.js is not appropriate, then use .NET Core
+- use PostgreSQL for relational databases
+- use CosmosDB for NoSQL databases
+- do not store secrets in a pod, instead use AAD Pod Identity
```

Comment:
I thought secrets were loaded into pods via `Secret` Kubernetes resources?
If the bullet read `do not store _credentials_ in a pod...` that would make more sense to me. Is that more accurate? I might be missing something though.

Reply:
I've deliberately not mentioned the `Secret` resource in Kubernetes because it's the subject of a bit of debate on how secure it is (in Defra).

But the point I was trying to make here is, if the use of Pod Identity would allow you to avoid putting a secret into the cluster then you need to do it.  eg, we should have database passwords in our cluster.  I've reworded it to make it clearer.